src/schemas/validation/schema.yaml

```diff
@@ -937,6 +945,27 @@ $defs:
         $ref: '#/$defs/specification-extensions'
         unevaluatedProperties: false
 
+      device-authorization:
+        type: object
+        properties:
+          deviceAuthorizationUrl:
+            type: string
+            format: uri-reference
+          tokenUrl:
+            type: string
+            format: uri-reference
+          refreshUrl:
+            type: string
+            format: uri
```

Comment:
Curious that this is a uri and not uri-reference.

Reply:
Thanks, it should be as all `refreshUrl`s are now in the [3.1 schema](https://github.com/OAI/OpenAPI-Specification/blob/v3.1-dev/src/schemas/validation/schema.yaml#L868-L870)